src/order/order.controller.spec.ts

```diff
@@ -0,0 +1,219 @@
+import { INestApplication } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import request from 'supertest';
+import { OrderController } from './order.controller';
+import { OrderService } from './order.service';
+import { ApplicationModule } from '~/application.module';
+import { TransferReceivedOrderBodyDTO } from './order.type';
+import { Role } from '~/entity/Role';
+import { Member } from '~/member/entity/member.entity';
+import { App } from '~/entity/App';
+import { AppPlan } from '~/entity/AppPlan';
+import { AppSetting } from '~/app/entity/app_setting.entity';
+import { OrderLog } from './entity/order_log.entity';
+import { EntityManager, Repository } from 'typeorm';
+import { AppSecret } from '~/app/entity/app_secret.entity';
+import { v4 } from 'uuid';
+import { sign } from 'jsonwebtoken';
+import { role, app, appPlan, appSecret, appSetting } from '~/../test/data';
+import { ConfigService } from '@nestjs/config';
+
+const apiPath = {
+  auth: {
+    token: '/auth/token',
+  },
+  order: {
+    transferReceivedOrder: '/orders/transfer-received-order',
+  },
+};
+
+describe('OrderController (e2e)', () => {
+  let application: INestApplication;
+  let configService: ConfigService;
+  let manager: EntityManager;
+  let roleRepo: Repository<Role>;
+  let appPlanRepo: Repository<AppPlan>;
+  let appRepo: Repository<App>;
+  let appSecretRepo: Repository<AppSecret>;
+  let appSettingRepo: Repository<AppSetting>;
+  let memberRepo: Repository<Member>;
+  let orderLogRepo: Repository<OrderLog>;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [OrderService],
+      controllers: [OrderController],
+      imports: [ApplicationModule],
+    }).compile();
+
+    application = module.createNestApplication();
+    configService = application.get<ConfigService<{ HASURA_JWT_SECRET: string }>>(ConfigService);
+    manager = application.get<EntityManager>('phdbEntityManager');
```

Comment:
`manager = application.get<EntityManager>('phdbEntityManager');`  

Cange to

`manager = application.get<EntityManager>(getEntityManagerToken('phdb'));`